hedera-mirror-rest-java/src/test/java/com/hedera/mirror/restjava/controller/AllowancesControllerIntegrationTest.java

```diff
@@ -72,7 +74,34 @@ void setup() {
 
     @Test
     void successWithNoQueryParamsShardRealmNumAccountId() {
```

Comment:
Since `AllowancesController` can in the future contain other allowances APIs, should prefix all tests with `nftAllowances`. Can also simplify names to not mention things that aren't present.

```suggestion
    void nftAllowancesEntityId() {
```